packages/@aws-cdk/aws-cloudfront/package.json

```diff
@@ -95,6 +95,7 @@
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "stability": "experimental",
+  "maturity": "experimental",
```

Comment:
This one feels a bit odd, because the `CloudFrontWebDistribution` is stable (see "features"). But I guess the stability/maturity are currently capturing the minimal (implemented) stability/maturity. In cases where one set of functionality is just not implemented, defaulting to the implemented feature's maturity makes sense; in cases where there are multiple features in varying degrees of maturity, it's unclear what we define the module's maturity as.

Reply:
There was an effort several months ago along the lines of "feature groups" but it was never finalized, and with the v2 push to graduate everything and separate experiments, maybe it's not worth diving into this.